++ set -o pipefail
++ export ECF_PORT=16889
++ ECF_PORT=16889
++ export ECF_HOST=login4.bullx
++ ECF_HOST=login4.bullx
++ export ECF_NAME=/DALES/postprocessing/store_somewhere
++ ECF_NAME=/DALES/postprocessing/store_somewhere
++ export ECF_PASS=31zqscDp
++ ECF_PASS=31zqscDp
++ export ECF_TRYNO=3
++ ECF_TRYNO=3
++ export ECF_RID=28534
++ ECF_RID=28534
++ export PATH=/usr/local/apps/ecflow/4.7.0/bin:/lustre3/repository/Packages/netcdf/netcdf-fortran-4.4.4/bin:/lustre3/repository/Packages/netcdf/netcdf-c-4.6.1/bin:.:/nfs/home/users/theeuwes/bin:/lustre3/repository/Packages/bin:/nfs/home/users/theeuwes/bin:/nfs/home/users/theeuwes/cmake-3.18.4/bin:/opt/intel/compilers_and_libraries_2017.2.174/linux/bin/intel64:/opt/intel/compilers_and_libraries_2017.2.174/linux/mpi/intel64/bin:/opt/intel/debugger_2017/gdb/intel64_mic/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/nfs/home/users/theeuwes/.local/bin:/nfs/home/users/theeuwes/bin
++ PATH=/usr/local/apps/ecflow/4.7.0/bin:/lustre3/repository/Packages/netcdf/netcdf-fortran-4.4.4/bin:/lustre3/repository/Packages/netcdf/netcdf-c-4.6.1/bin:.:/nfs/home/users/theeuwes/bin:/lustre3/repository/Packages/bin:/nfs/home/users/theeuwes/bin:/nfs/home/users/theeuwes/cmake-3.18.4/bin:/opt/intel/compilers_and_libraries_2017.2.174/linux/bin/intel64:/opt/intel/compilers_and_libraries_2017.2.174/linux/mpi/intel64/bin:/opt/intel/debugger_2017/gdb/intel64_mic/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/nfs/home/users/theeuwes/.local/bin:/nfs/home/users/theeuwes/bin
++ ecflow_client --init=28534
++ trap ERROR 0
++ trap '{ echo "Killed by a signal"; ERROR ; }' 1 2 3 4 5 6 7 8 10 12 13 15
++ /nfs/home/users/theeuwes/work/DALES_runs/ecf/scr/store_DALES_output.py 20220523 1
Copying : /nfs/home/users/theeuwes/work/DALES_runs/20220523/profiles.001.nc
Traceback (most recent call last):
  File "/nfs/home/users/theeuwes/work/DALES_runs/ecf/scr/store_DALES_output.py", line 32, in <module>
    shutil.copy(i,j)
  File "/usr/lib64/python3.6/shutil.py", line 245, in copy
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/usr/lib64/python3.6/shutil.py", line 120, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: '/nfs/home/users/theeuwes/work/DALES_runs/20220523/profiles.001.nc'
++ ERROR
++ set +e
++ wait
++ ecflow_client --abort=trap
++ trap 0
++ exit 0
